File: Golang/0002.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	input, _ := reader.ReadString('\n')
	input = strings.TrimSpace(input)

	numbers := strings.Split(input, " ")
	a, _ := strconv.Atoi(numbers[0])
	b, _ := strconv.Atoi(numbers[1])

	var x string
	if a > b {
		x = ">"
	} else if a == b {
		x = "="
	} else {
		x = "<"
	}

	fmt.Println(x)
}

File: Dart/0002.dart
import 'dart:io';

void main() {
  String input = stdin.readLineSync()!;
  List<String> numbers = input.split(' ');

  int a = int.parse(numbers[0]);
  int b = int.parse(numbers[1]);

  String x;
  if (a > b) {
    x = '>';
  } else if (a == b) {
    x = '=';
  } else {
    x = '<';
  }

  print(x);
}